### PR TITLE
feat(nimbus): Sync VPN FML

### LINKED ISSUE #9969: Sync VPN fml to experimenter
VPN FML is available at [https://github.com/mozilla-services/guardian-website/blob/main/nimbus.yaml](https://github.com/mozilla-services/guardian-website/blob/main/nimbus.yaml|smart-link)  sync the fml to experimenter so that VPN should be able to get the features list

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4112)

### PR DESCRIPTION
**Pull request by yashikakhurana**
Because

- VPN FML is [available](https://github.com/mozilla-services/guardian-website/blob/main/nimbus.yaml) and we want to sync with the experimenter so that the experiment owner can select a feature for VPN

This commit

- Add new sync support for FML

Fixes #9969

### CODE REVIEW

**Comment by yashikakhurana** [comment]
block untile this get released- https://github.com/mozilla/experimenter/pull/9939

**Comment by brennie** [comment]
This is also blocked on private repo support in nimbus cli

File: Makefile [hunk 1]
```diff
@@ -52,6 +52,7 @@ FIREFOX_ANDROID_REPO = @mozilla-mobile/firefox-android
 FIREFOX_IOS_REPO     = @mozilla-mobile/firefox-ios
 FOCUS_IOS_REPO       = @mozilla-mobile/focus-ios
 MONITOR_REPO         = @mozilla/blurts-server
+VPN_REPO             = @mozilla-services/guardian-website
```
Comment (brennie) [inline Makefile:55]: This isn't needed. These were only used before manifest tool did fetches. All of these constants can be removed.

Comment (yashikakhurana) [reply Makefile]: done, removed all of them

**Comment by brennie** [comment]
PR is up for nimbus-fml support here: https://github.com/mozilla/application-services/pull/6021

File: Makefile [hunk 2]
```diff
@@ -48,11 +48,6 @@ PYTHON_PATH_SDK = PYTHONPATH=/application-services/megazord
 
 JETSTREAM_CONFIG_URL = https://github.com/mozilla/metric-hub/archive/main.zip
 
-FIREFOX_ANDROID_REPO = @mozilla-mobile/firefox-android
-FIREFOX_IOS_REPO     = @mozilla-mobile/firefox-ios
-FOCUS_IOS_REPO       = @mozilla-mobile/focus-ios
-MONITOR_REPO         = @mozilla/blurts-server
```
Comment (eliserichards) [inline Makefile:54]: Why are these being removed in this PR? Can they be removed separately?

Comment (yashikakhurana) [reply Makefile:54]: oh so they are not being used, suggested to remove - https://github.com/mozilla/experimenter/pull/9970#discussion_r1432091055

Comment (eliserichards) [reply Makefile:54]: Ahh, perfect! 👍

**Review by eliserichards — APPROVED** [review]
Lookin' good!

**Review by brennie — APPROVED** [review]

File: experimenter/experimenter/features/manifests/apps.yaml [hunk 3]
```diff
@@ -114,3 +114,10 @@ focus_ios:
 #           - "beta"
 #           - "release"
 #           - "esr115"
+
+mozillavpn_backend_cirrus:
+  slug: "vpn-web"
+  repo:
+    type: "github"
+    name: "mozilla-services/guardian-website"
+  fml_path: "nimbus.yaml"
```
Comment (jaredlockhart) [inline experimenter/experimenter/features/manifests/apps.yaml:123]: 🎉 🎉 🎉 🎉 🎉 🎉

**Comment by yashikakhurana** [comment]
Since we added `production` as the channel option, but in VPN repo it says `vpn`, I am working on making a update, once they update, we can merge this PR, Adding a `block` label for now

**Comment by yashikakhurana** [comment]
waiting on this- https://github.com/mozilla-services/guardian-website/pull/1690

**Pull request merged**